src/components/biggive-main-menu/biggive-main-menu.tsx

```diff
@@ -141,5 +152,206 @@ export class BiggiveMainMenu {
         </nav>
       </Host>
     );
+
+    /**
+     * @todo take prop passed in for base domain or environment (e.g. dev|staging|regression|production ) so we can
+     *       output appropriate nav links to stay in same envirionment.
+     */
+    const presetContentMenu = () => {
```

Comment:
Content bellow is copied from https://github.com/thebiggive/donate-frontend/blob/004a6d9b6f631726a0597536eabd400195fb5b7d/src/app/app.component.html#L1 and then tweaked slightly to work outside Angular.